src/core-layers/hexagon-cell-layer/hexagon-cell-layer.js

```diff
@@ -73,9 +72,8 @@ export default class HexagonCellLayer extends Layer {
   }
 
   getShaders() {
-    return enable64bitSupport(this.props)
-      ? {vs: vs64, fs, modules: ['project64', 'lighting', 'picking']}
-      : {vs, fs, modules: ['lighting', 'picking']}; // 'project' module added by default.
+    const projectModule = enable64bitSupport(this.props) ? 'project64' : 'project32';
```

Comment:
Nit: `enable64bitSupport` sounds like it is changing some state. Should this be named `fp64enabled`, or `is64bitEnabled`?

i also think it might be better as a method on `Layer` as this function will need to be exported from `core` to `core-layers`.

Reply:
Will address in a follow up PR.